lumigator/frontend/e2e/vue.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { test, expect } from '@playwright/test';
+
+// See here how to get started:
+// https://playwright.dev/docs/intro
+test('visits the app root url', async ({ page }) => {
+  await page.goto('/');
+  await expect(page.locator('div.greetings > h1')).toHaveText('You did it!');
+})
```

Comment:
❓Please forgive my ignorance, but is this just an example test or should it actually pass? (should we remove it if not).

Reply:
Yes, its only an example test that is failing in that case, it was there to test running the e2e tests, I can potentially replace it with a sample running test that applies to our application instead